apache/trafficcontrol
# TP: upgrades application build dependencies

## What does this PR (Pull Request) do?

Upgrades TP's application dependencies. No documentation updates required and current UI tests are sufficient to ensure no regressions.


## Which Traffic Control components are affected by this PR?

- Traffic Portal

## What is the best way to verify this PR?

1. Start up cdn in a box which will build and package TP into an RPM. This RPM will then be installed and the Traffic Portal service will be started. Ensure TP runs without errors.

2. Run the [UI tests](https://github.com/apache/trafficcontrol/tree/master/traffic_portal/test/end_to_end) pointed at TP in cdn in a box to ensure any existing functionality was not broken.

## The following criteria are ALL met by this PR
